tests/base/src/test/java/org/wildfly/security/http/digest/DigestAuthenticationMechanismTest.java

```diff
@@ -60,6 +60,17 @@ public static void removePasswordProvider() {
         Security.removeProvider(provider.getName());
     }
 
+    public void testDigest(String []authorization,HttpServerAuthenticationMechanism mechanism) throws Exception{
+        TestingHttpServerRequest request = new TestingHttpServerRequest(authorization);
+        mechanism.evaluateRequest(request);
+        Assert.assertEquals(Status.COMPLETE, request.getResult());
+    }
+    public void testDigest(String []authorization,HttpServerAuthenticationMechanism mechanism,String uri) throws Exception{
```

Comment:
@keshav-redhat Just a minor, missing space between the new methods and between parameters, same below